print/server-middleware/routes/test3.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Browserless.io puppeteer endpoint (for more granular control)
+ * HTML generated via separate call from Browserless.io to our Nuxt Print App
+ */
+
+import puppeteer from 'puppeteer'
+const { Router } = require('express')
+
+const router = Router()
+
+// Test route
+router.use('/test3', async (req, res) => {
+  // Launch own puppeteer + Chromium
+  // const browser = await puppeteer.launch()
+
+  // Connect to browserless.io (puppeteer websocket)
+  const browser = await puppeteer.connect({
+    browserWSEndpoint: `wss://chrome.browserless.io/?token=${process.env.BROWSERLESS_TOKEN}`,
+  })
+
+  const page = await browser.newPage()
+
+  try {
+    // HTTP request back to Print App
+    // TODO: here we need to provide some sort of authentication (pass JWT)
```

Comment:
You can pass the JWT token in the Authorization header instead of sending the cookies to the user:
```
Authorization: Bearer <contents-of-jwt_hp-cookie>.<contents-of-jwt_s-cookie>
```
You can set the header on the puppeteer page before fetching the URL: https://stackoverflow.com/a/63062387